src/date/__tests__/monthsAfterStr-test.js

```diff
@@ -0,0 +1,10 @@
+import { expect } from 'chai';
+import monthsAfterStr from '../monthsAfterStr';
+
+describe('monthsAfterStr', () => {
+    it('should return a date string with given number of months ahead of the current date', () => {
```

Comment:
then name it to `xMonthsAfter` then, I think conversion to str can be called by user, we do not have to dictate this.

As for test, test serves as documentation, I know it works, but we should document that it works